chase/frontend/components/login/password.tsx

```diff
@@ -88,7 +89,7 @@ export default function usernameLogin({ changeLoginState }) {
             }}
           >
             Ich bin Einverstanden mit den{" "}
-            <a
+            <Link
               href="/terms"
               target="_blank"
               rel="noreferrer"
```

Comment:
What do these do?

Reply:
target: opens the window in a new tab (since the privacy and terms pages should not hinder the login flow
rel: I have no idea. Rome told me it was good practise when still using the <a> tag and I decided to keep just in case. The <Link> Component is based on <a> as far as I read, so that all properties should also work with Link

Reply:
Ah ok. Just looked it up, rel prevents the browser from telling the page behind the link where the user is coming from. Privacy wise I wonder why this even is a thing and not disabled by default...